kymatio/scattering1d/core/timefrequency_scattering.py

```diff
@@ -268,3 +276,45 @@ def frequency_scattering(X, backend, filters_fr, oversampling_fr,
         # and n=(n2, n_fr) if spinned. This 'n' tuple is unique.
         yield {**X, 'coef': Y_fr, 'n': (X['n'][1:] + (n_fr,)),
             'j_fr': j_fr, 'n_fr': n_fr, 'spin': spin}
+
+
+def time_averaging(U_2, backend, phi_f, oversampling):
+    """
+    Parameters
+    ----------
+    U_2 : dictionary with keys 'coef' and 'j', typically returned by
+        frequency_scattering
+    backend : module
+    phi_f : dictionary. Temporal low-pass filter in Fourier domain,
```

Comment:
Would the `phi_f` here introduce confusion with the frequential low-pass filter?

Reply:
agree, is this a typo?

Reply:
This is not a typo. The `_f` means fourier domain, but it's still associated to the time variable, not the log-frequency variable. This is for consistency with scatteringd1d

for frequential scattering i use the suffix `_fr`.
See `J_fr`, `Q_fr`, `_N_padded_fr`, `filters_fr`

So the frequential filters have both suffixes: `phi_fr_f` and `psi_fr_f`

Reply:
Make sense! Documenting this in the code may also help the user.

Reply:
these docstrings in core are more "developer's notes" (aimed at people like cyrus / lily / muawiz / you / etc.) than user's notes per se. Ordinary users do not call core Kymatio functions directly but via frontends instead.

This is why core/scattering1d.py does not have a docstring (yet). But for JTFS, the core algorithm is fairly complicated and has several functions so i figured it would be helpful for maintainers and reviewers to have these docstrings as self-help.

Reply:
Good point. Thank you!